Libplanet/Blockchain/FungibleAssetStateCompleters.cs

```diff
@@ -40,6 +41,9 @@ public static class FungibleAssetStateCompleters<T>
             Currency currency
         ) =>
             (blockChain, hash) =>
-                (Bencodex.Types.Integer)stateCompleter(blockChain, hash, address, currency);
+            {
+                FungibleAssetValue balance = stateCompleter(blockChain, hash, address, currency);
+                return (Bencodex.Types.Integer)balance.RawValue;
```

Comment:
Is it okay returning only `.RawValue` without `currency`?

Reply:
Yes, raw state completer (`Func<BlockChain<T>, HashDigest<SHA256>, IValue>`) used by only `BlockChain<T>.GetRawState()` method, which is `internal`, takes `IValue` instead of `FungibleAssetValue`.